src/components/Form/constants.js

```diff
@@ -0,0 +1,13 @@
+import PropTypes from 'prop-types';
+
+export const PICK_LIST_SIZE = {
+  SMALL: 'small',
```

Comment:
These are pre-defined slds terms, no need to hide them behind constants.

Reply:
`PICK_LIST_SIZE` itself is a constant to hide slds internals.

We will use it as:
```
<Picklist
...
size={Picklist.size.SMALL}
```